### DISCUSSION TITLE
Recette v8.3.0

### DISCUSSION THREAD

**Discussion by alexisig** [MTES-MCT/sparte#703] [Recettes]
Environnement de test : https://sparte-staging-pr652.osc-fr1.scalingo.io/

----

Pour tester : 
- Vérifier que les [SCOTs](https://scotenaction.fedescot.org/visualiser/explorer#layers=3d456eaf82e25715db7f9d451e6585ad) sont diagnosticables. Les données peuvent être différentes de celles présentées en production pour un même SCOT car l'emprise des SCOTs a pu changer (prod : COG 2022, app review : COG 2024)
- Vérifier que les [EPTs](https://fr.wikipedia.org/wiki/%C3%89tablissement_public_territorial) sont diagnosticables
- Vérifier que les diagnostics existant fonctionnent toujours.
- Vérifier que les diagnostics des territoires ayant subi des changements de COG fonctionnent. Plusieurs cas de figure : 
    - Plusieurs communes ont été fusionnées. Toutes les données de conso sont affichées, mais un message d'information sur la fusion est affichée. Communes concernées : 08053, 16097, 18173, 25060, 35062, 49160, 64300, 69149, 85292, 86247, 95169.
    - Plusieurs communes ont été divisées. Les données de conso ne sont pas affichées à la commune, un message est affiché pour informer l'utilisateur, les téléchargements ne sont pas disponibles. Les informations sont toujours disponibles aux mailles supérieures (EPCI et/ou SCOT). Communes concernées : 85084, 85165, 85212, 60054, 60694
    - Plusieurs communes sont incorrectement sur un ancien COG dans les données du Cerema (absence de division). Le message d'information est différent que pour la division, mais le comportement est le même. Communes concernées : 14712, 14666, 52465, 52033, 52332, 52504, 52124, 52031, 52278, 55298, 55138, 76676, 76601
    - Quelques communes manquantes dans les données du Cerema ont été initialisées à 0. Un message s'affiche pour l'utilisateur. Les téléchargements ne sont pas disponibles. Communes concernées : 09304, 29083, 29084
    - Vérifier que les diagnostics bloqués en production fonctionnent sur l'app review.

**Comment by leabouche** [comment]
Les pages conso des SCoTs ne chargent pas (testé sur plusieurs SCoTs) - erreur : 
![erreur loading page conso scot ](https://github.com/user-attachments/assets/826ab691-ca63-45cf-91c1-a6fb8582bcd5)

**Reply by alexisig** [reply]
corrigé

**Comment by leabouche** [comment]
Les territoires de comparaison des EPTs ne sont pas que des EPTs. Il me semble que pour les autres mailles on ne sélectionne que des mailles de meme catégorie. 
![territoires comparaison EPT](https://github.com/user-attachments/assets/e624d846-f98a-42f2-b84d-eaa408ca8b3a)

**Reply by alexisig** [reply]
Bien vu. Par défaut c'est le fonctionnement attendu (c'est comparé avec EPCI voisins, les EPT étant des EPCI).
Je rajoute en dur la comparaison de tous les EPT avec les autres EPT du coup 👍

**Reply by alexisig** [reply]
temporairement retiré les EPT

**Comment by leabouche** [comment]
Affichage des départements avec l'OCGGE sur la page d'accueil ? 
![depts ocsge](https://github.com/user-attachments/assets/2a593b98-868b-411f-b45c-13c5f23e95a4)

**Reply by alexisig** [reply]
yes c'est normal, on a que les données de la Gironde sur cet environnement

**Comment by leabouche** [comment]
typo sur le message affiché pour les communes ayant été divisées au nouveau COG - "ayant été divisée" --> "ayant été divisé". 
![typo](https://github.com/user-attachments/assets/4cf7b9c3-97bf-430f-abf0-fba9bb147dce)

**Reply by alexisig** [reply]
corrigé

**Comment by leabouche** [comment]
Diagnostics testés et non reçus pour le moment : 
- Vendée (conso et rapport triennal)
- PACA (rapport triennal)
- Rhône (analyse complète)

**Reply by leabouche** [reply]
reçu PACA

**Reply by leabouche** [reply]
reçu les deux autres avec le fix :)

**Reply by alexisig** [reply]
top ! 
Pour le PACA c'est super long, je regarde ce qu'il est possible de faire

**Reply by leabouche** [reply]
ça a mis un peu moins d'une heure (j'ai pas mis le message au moment ou j'ai recu le rapport)

**Reply by leabouche** [reply]
ca reste long :)

**Reply by alexisig** [reply]
corrigé également, ça devrait tourner en moins d'une minute 👍
